docs/docs/run/4_networking.mdx

```diff
@@ -112,8 +112,8 @@ Currently, only services of type `http` can be exposed to the internet.
 
 2. Once the service of type `http` is published, you can connect to it using a URL with the following format:
 
-  `https://<subdomain>-<svc_name>-<port>.svc.<region>.nhost.run`
+  `https://<run_subdomain>-<port>.svc.<region>.nhost.run`
```

Comment:
Shouldn't this be `service_subdomain`?

Reply:
I think it should be either run or run_service, service alone is ambiguous I think.